src/Ledger/PParams.lagda

```diff
@@ -21,14 +21,13 @@ module Ledger.PParams
   (es     : _) (open EpochStructure es)
   (ss     : ScriptStructure crypto es) (open ScriptStructure ss)
   where
+ProtVer : Set
+ProtVer = ℕ × ℕ
```

Comment:
`ProtVer` is retired, so it seems there's no reason to include it in the figure.

Reply:
The type `ProtVer` isn't retired, just the protocol parameter. It's still used in the `TriggerHF` GA and `EnactState`.

Reply:
...but is it okay to leave `ProtVer` out of the figure, as long as it's still documented in the prose?

Reply:
At some point I'd like to move `ProtVer` to some other figure where it makes more sense, but you don't have to do it now